test/integration/bun-types/fixture/serve-types.test.ts

```
@@ -20,13 +20,18 @@ export default {
       expectType(ws.data).is<{ name: string }>();
     },
   },
-} satisfies Bun.ServeOptions<{ name: string }>;
+  routes: {
+    "/": req => {
+      expectType(req.params).is<Record<string, string>>();
+    },
+  },
+} satisfies Bun.Serve.Options<{ name: string }>;
 
 function expectInstanceOf<T>(value: unknown, constructor: new (...args: any[]) => T): asserts value is T {
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)